analytics_data_api/v0/tests/views/__init__.py

```diff
@@ -12,3 +15,27 @@ def setUpClass(cls):
         cls.course_id = DEMO_COURSE_ID
         cls.course_key = CourseKey.from_string(cls.course_id)
         super(DemoCourseMixin, cls).setUpClass()
+
+
+class VerifyNoCourseIdMixin(object):
```

Comment:
Why not add both these `verify` methods to the same mixin? Perhaps the `VerifyCourseIdMixin`?

In general, I think that if a mixin has only a single method, that method might as well be a function not bound to an object or class.